### DISCUSSION TITLE
Does JobRunr support Active -Active Data centre

### DISCUSSION THREAD

**Discussion by abhiram100906** [jobrunr/jobrunr#310] [Q&A]
Hi,

We are currently evaluating Jobrunr and would like to know the behaviour of the framework for the following scenarios :

**Scenario : Active - Active Data Centre with peer to peer data replication**

1. Lets imagine we have two Data centers DC1 & DC2 
2. JobRunr instance JR1 in DC1 and JobRunr instance JR2 in DC2
3. JobRunr JR1 will insert a Job record into Active Node of DC1 DB Cluster
4. Data base replication takes place and the record is now copied into the other DC2

Is there any possibility for race condition where in JR1 and JR2 can pick the same Job for processing ? 

Please do let us know.

Thanks,
Abhiram

**Comment by abhiram100906** [comment]
@rdehuyss would appreciate if you could address the above concern.

**Comment by rdehuyss** [comment]
No, I'm afraid that won't work. The DB is the single source of truth and as you are duplicating it there are multiple sources of truth. JR1 and JR2 can pick up the same job.

**Reply by abhiram100906** [reply]
thanks @rdehuyss
